pydis_core/utils/interactions.py

```diff
@@ -80,6 +80,12 @@ async def interaction_check(self, interaction: Interaction) -> bool:
         await interaction.response.send_message("This is not your button to click!", ephemeral=True)
         return False
 
+    async def stop(self) -> None:
```

Comment:
Will the fact that this is async break any existing usage of this class? The regular discord.py `discord.ui.View` class's `.stop()` method is sync, so anything using this class's `stop` method before this change would be sync.